### PR TITLE
Database implementation

### PR DESCRIPTION
**Pull request by jonathanhkim**

### CODE REVIEW

**Review by AstonKhor — APPROVED** [review]
Looks good. I liked that you cleared the database using deleteMany.
I think that you'd want to have more seeding data(100 points).

File: server/db/seed.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+const companyNames = require('./templates/companyNames');
+const dataCreate = require('./templates/dataCreate');
+
+const createObjects = () => {
+  let currentStockPrice = Math.floor(Math.random() * 100);
+  let graphData  = [];
+
+  // integer that is the current price
+  console.log("creating new collections...")
+  Object.entries(companyNames).forEach( (entry, idx) => {
```
Comment (kwillard95) [inline server/db/seed.js:13]: I like how you used Object.entries! I have never seen this before.

**Review by NickHolke — APPROVED** [review]
Looks good. Like Aston said, potentially add some more companies.

File: server/db/templates/dataCreate.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const createData = (currentValue) => {
+  const data = []
+  let dates = ["Q1 2018","Q2 2018","Q3 2018","Q4 2018","Q1 2019","Q2 2019","Q3 2019","Q4 2019"];
+  dates.forEach((date, idx) => {
+    data.push({
+      date: date,
+      estimated: (Math.random() * currentValue),
+      actual: (Math.random() * currentValue),
+    })
```
Comment (kwillard95) [inline server/db/templates/dataCreate.js:9]: Why not use .map here instead?

File: server/db/seed.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    let newGraph = {
+      company: entry[0],
+      title: entry[1],
+      scale: "",
+      data: dataCreate(currentStockPrice)
+    }
+
+    console.log(`Companies: ${idx + 1}/${Object.keys(companyNames).length}`)
```
Comment (kwillard95) [inline server/db/seed.js:21]: I really like how you use console.logs to keep track of where you're at in the seeding process

**Review by kwillard95 — APPROVED** [review]

File: server/db/templates/dataCreate.js (hunk 2 above)
Comment (jonathanhkim) [reply server/db/templates/dataCreate.js:9]: I use forEach instead because map creates a new array. forEach just iterates over the current one.

**Pull request merged**